lib/api/definitions/__tests__/CreatedMessage.test.ts

```diff
@@ -0,0 +1,261 @@
+import {
+  CreatedMessage,
+  isCreatedMessage,
+  toCreatedMessage
+} from "../CreatedMessage";
+
+import { toMessageBodyMarkdown } from "../MessageBodyMarkdown";
+
+import { MessageContent } from "../MessageContent";
+
+import { toMessageSubject } from "../MessageSubject";
+
+import { toFiscalCode } from "../FiscalCode";
+
+import { toTimeToLive } from "../TimeToLive";
+
+describe("Check CreatedMessage methods", () => {
```

Comment:
you should have a `describe` block for each method (you're testing each method)

then you should have a `test()` or `it()` block for each test you want to execute

for instance

```
describe("CreatedMessage#toCreatedMessage", () => {
  it("should return a defined option for valid payloads", () => {
  });

  it("should return an empty option for invalid payloads", () => {
  });
});
```